Sequencer.cpp

```diff
@@ -0,0 +1,57 @@
+#include "Sequencer.h"
+
+Sequencer::Sequencer()
+{
+    clockInputVolts = 4; seqLength = 4; inputGateIsHigh = true; rangeSemitones = 0; rangeLengthInputVolts = 12.f; randomInputIsHigh = true; maxSemitoneRange = 18; noteLengthMs = 500;
+}
```

Comment:
use the member initializer list.
Don't initialize member variables inside constructor bodies.
- [ ] revise